api/nodes/serializers.py

```diff
@@ -287,7 +287,8 @@ class NodeSerializer(TaxonomizableSerializerMixin, JSONAPISerializer):
 
     forks = RelationshipField(
         related_view='nodes:node-forks',
-        related_view_kwargs={'node_id': '<_id>'}
+        related_view_kwargs={'node_id': '<_id>'},
+        related_meta={'count': 'get_fork_count'}
```

Comment:
@hmoco added this in `develop` already: https://github.com/CenterForOpenScience/osf.io/pull/8370/files#diff-7a6d21a50624153d15e63a5f643401c7R291